src/tests/ftest/recovery/pool_membership.py

```diff
@@ -295,3 +303,96 @@ def test_dangling_pool_map(self):
             errors.append(msg)
 
         report_errors(test=self, errors=errors)
+
+    def test_dangling_rank_entry(self):
+        """Test dangling target entry.
+
+        1. Create a pool and a container.
+        2. Write some data with IOR using SX.
+        3. Stop servers.
+        4. Remove pool directory from one of the mount points.
+        5. Enable checker.
+        6. Start checker.
+        7. Query the checker until expected number of inconsistencies are repaired.
+        8. Disable checker and start servers.
+        9. Wait for rebuild to finish.
+        10. Query the pool and verify that expected number of targets are disabled.
+
+        Jira ID: DAOS-11735
+
+        :avocado: tags=all,pr
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pool_membership
+        :avocado: tags=PoolMembershipTest,test_dangling_rank_entry
+        """
+        targets = self.params.get("targets", "/run/server_config/engines/0/*")
+        exp_msg = "dangling rank entry"
+
+        # 1. Create a pool and a container.
+        # self.log_step("Create a pool and a container.")
+        self.pool = self.get_pool(connect=False)
+        self.container = self.get_container(pool=self.pool)
+
+        # 2. Write some data with IOR using SX.
+        self.ior_cmd.set_daos_params(self.server_group, self.pool, self.container.uuid)
```

Comment:
We should use labels unless there is a specific need
```suggestion
        self.ior_cmd.set_daos_params(self.server_group, self.pool, self.container.identifier)
```

Reply:
Done. I copied and pasted and forgot to update it.